packages/expo-image-picker/src/ImagePicker.types.ts

```diff
@@ -261,6 +261,9 @@ export type ImagePickerAsset = {
    * The `exif` field is included if the `exif` option is truthy, and is an object containing the
    * image's EXIF data. The names of this object's properties are EXIF tags and the values are the
    * respective EXIF values for those tags.
+   *
+   * @platform ios
+   * @platform android
```

Comment:
```suggestion
   * @platform android
   * @platform ios
```

As per our writing styleguide, we prefer to starting Platform order from Android when referencing multiple platforms: https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#platform-order-when-referencing-android-ios-and-web-or-multiple-platforms

Can you please swap their order?

Reply:
Ops, sorry about that. I copied it directly from another property in the same file. Would you like to change this at the same time?
<img width="804" alt="image" src="https://github.com/user-attachments/assets/f0cb9918-6bac-4331-a0fc-2a38edc6043b">

Reply:
I swapped the order of supported platforms of the `assetId` property as well while at it 👍